baseplate/thrift/baseplate.thrift

```diff
@@ -127,3 +127,161 @@ struct Request {
     5: OriginService origin_service;
     6: Geolocation geolocation;
 }
+
+/** The integer values within this enum correspond to HTTP status codes.
+
+HTTP layers can easily map errors to an appropriate status code.
+*/
+enum ErrorCode {
+    /** This is appropriate if the client sent an invalid request.
```

Comment:
Since docstrings in the Thriftfile can be turned into docstrings in the generated code, I think we might want to consider the developer who got this error message in respond to an RPC they made the audience for the docs here. Given that, "This indicates..." would be more clear than "This is appropriate...".